_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
According to Josh's suggestion, we must have a reference for the 1.9 Release Notes. Make sure we are always referring to the page proposed in https://github.com/kubeflow/website/pull/3799 (planned to be merged right before the 1.9 Release date)